preferencebundle/Resources/ar.lproj/About.strings

```diff
@@ -0,0 +1,17 @@
+{
+    "ABOUT_TITLE" = "\U0627\U0644\U0645\U0632\U064a\U062f ";
+    "BUILD_DATE" = "\U062a\U0627\U0631\U064a\U062e \U0628\U0646\U0627\U0621 \U0627\U0644\U0627\U062f\U0627\U0629 ";
+    "BYPASS_VERSION" = "\U0625\U0635\U062f\U0627\U0631 \U062a\U062e\U0637\U064a \U0627\U0643\U062a\U0634\U0627\U0641 \U0627\U0644\U062c\U064a\U0644\U0628\U0631\U064a\U0643 ";
+    "CREDITS_HDR" = "\U0627\U0644\U0645\U0633\U0627\U0647\U0645\U064a\U0646 \U0641\U064a \U0627\U0644\U062a\U0637\U0648\U064a\U0631 ";
+    DEVELOPER = "\U0627\U0644\U0645\U0637\U0648\U0631";
+    "LATEST_VERSION" = "\U0627\U062d\U062f\U062b \U0625\U0635\U062f\U0627\U0631 ";
+    "LOCALIZATION_HDR" = "\U062a\U0631\U062c\U0645\U0629";
+    "PACKAGE_VERSION" = "\U0625\U0635\U062f\U0627\U0631 \U0627\U0644\U0627\U062f\U0627\U0629 ";
+    "SERVICE_VERSION" = "\U0625\U0635\U062f\U0627\U0631 \U0627\U0644\U062e\U062f\U0645\U0629";
+    "SOFTWARE_LICENSE" = "\U062a\U0631\U062e\U064a\U0635 \U0627\U0644\U0628\U0631\U0646\U0627\U0645\U062c";
+    TRANSLATOR = jjolano;
```

Comment:
update this with your name, because it will show in the About section 👍

Reply:
Now it’s okay?